plugins/git/webgui/js/gitMenu.js

```diff
@@ -30,8 +31,37 @@ function (topic, Menu, MenuItem, PopupMenuItem, model, viewHandler) {
     }
   };
 
-  viewHandler.registerModule(team, {
+  viewHandler.registerModule(nodeMenu, {
     type     : viewHandler.moduleType.TextContextMenu,
     priority : 70
   });
+
+  var fileMenu = {
+    id : 'git-text-team-file',
+    render : function (fileInfo) {
+      if (fileInfo.type === "Dir")
+        return;
+
+      var submenu = new Menu();
+
+      submenu.addChild(new MenuItem({
+        label : 'Blame',
```

Comment:
I know this might be a bit too much political correctness but this is the part we could rename "Blame" into something less negative. GitHub calls it "History" or "recent changes". :slightly_smiling_face:

Reply:
I think we should use the Git conventional names. If it's "blame" there then we should follow it. If they rename it, we can rename it too.

Reply:
I agree with @bruntib .